patches/graphql-tools+4.0.4.patch

```diff
@@ -0,0 +1,38 @@
+patch-package
+--- a/node_modules/graphql-tools/dist/stitching/schemaRecreation.js
++++ b/node_modules/graphql-tools/dist/stitching/schemaRecreation.js
+@@ -62,7 +62,9 @@ function recreateType(type, resolveType, keepResolvers) {
+         var newValues_1 = {};
+         values.forEach(function (value) {
+             newValues_1[value.name] = {
+-                value: value.value,
++                // Simply pass-through the name of the enum entry such that the enum type in the original schema will
++                // receive that and will be able to do its work as if it were never proxied.
++                value: value.name,
```

Comment:
@freiksenet Reverting this to [how you fixed it](https://github.com/apollographql/graphql-tools/commit/c500fdf82773b117b59a27aeffd3a90f22cd3a6d#diff-d5dcbf958f513509b72e378b5a9c0178R91) at the time did indeed fix the issue entirely 👍 

Once this current work is out we'll figure out a failing test upstream and perhaps the fix.